spec/lucky/action_spec.cr

```diff
@@ -82,6 +82,17 @@ class PlainText::Index < Lucky::Action
   end
 end
 
+class OptionalParams::Index < Lucky::Action
+  param page : Int32?
+  param with_default : String? = "default"
+  param with_int_default : Int32? = 1
+  param with_int_never_nil : Int32 = 1337
```

Comment:
Could you add a case for `param required_page : Int32`?

Reply:
✅